crypto/http/http_client.c

```diff
@@ -747,7 +747,7 @@ static ASN1_VALUE *BIO_mem_d2i(BIO *mem, const ASN1_ITEM *it)
     return resp;
 }
 
-static BIO *OSSL_HTTP_REQ_CTX_transfer(OSSL_HTTP_REQ_CTX *rctx)
+static BIO *ossl_http_req_ctx_transfer(OSSL_HTTP_REQ_CTX *rctx)
```

Comment:
curious why you kept the ossl prefix since it's a static function to this file.

Reply:
Well, not changing the length of a function name has its advantages :wink:
and if we should ever make the function public it would be a little simpler this way.